webknossos/webknossos/dataset/dataset.py

```diff
@@ -1173,6 +1174,10 @@ def add_layer_from_images(
                     chunk_shape = DEFAULT_CHUNK_SHAPE.with_z(1)
                 if chunks_per_shard is None:
                     chunks_per_shard = DEFAULT_CHUNKS_PER_SHARD_ZARR.with_z(1)
+
+            if chunks_per_shard is None and layer.data_format == DataFormat.Zarr3:
+                chunks_per_shard = DEFAULT_CHUNKS_PER_SHARD_FROM_IMAGES
+
```

Comment:
In my opinion, this should happen in `Layer.add_mag` method because the default values for `chunks_per_shard` for wkw and Zarr are set there.

Reply:
This is a new constant specifically for datasets that are converted from image sequences. I think it should be set in `from_images`. Do you agree?

Reply:
Yes, I agree then. What is the reason for using (128,128,1) as `chunks_per_shard` for `from_images` and (32,32,32) for other Zarr3 datasets?

Reply:
To efficiently write Zarr data, shards need to be written in one go. `from_images` typically is used with image sequences where we convert in batches of 32 (our chunk shape). The default shard shape of 1024 \** 3 would, therefore, not be efficient. I came up with 128,128,1 because that 16k chunks in one shard, which is similar to the 32k chunk in our default shards.